docs/sources/flow/get-started/install/kubernetes.md

```diff
@@ -20,20 +20,34 @@ weight: 200
 
 # Deploy {{% param "PRODUCT_NAME" %}} on Kubernetes
 
-{{< param "PRODUCT_NAME" >}} can be deployed on Kubernetes by using the Helm chart for {{< param "PRODUCT_ROOT_NAME" >}}.
+You can deploy {{< param "PRODUCT_NAME" >}} on Kubernetes using our Helm chart.
 
-## Before you begin
+## Dedicated guides
+
+We recommend using our dedicated guides for each telemetry signal to deploy a 
+production-ready {{< param "PRODUCT_NAME" >}} on Kubernetes:
+
+* For metrics, follow our dedicated [collect Prometheus metrics on Kubernetes][collect-prometheus] guide.
+* For logs, follow our dedicated [collect logs on Kubernetes][collect-logs] guide.
+* For anything else, follow the [generic Helm installation](#generic-helm-installation) instructions below.
```

Comment:
In the future we should have content to cover other telemetry signals, so fewer users would go to the 'generic helm installation'.